src/compiler/rust.rs

```diff
@@ -1131,7 +1131,7 @@ fn parse_arguments(arguments: &[OsString], cwd: &Path) -> CompilerArguments<Pars
                 cannot_cache!(concat!("missing ", stringify!($x)));
             };
         };
-    };
+    }
```

Comment:
Let's resolve clippy/compiler warnings in a separate patch, as they're unrelated to the MSVC flag shenanigans we're up to here.